torch/fx/experimental/symbolic_shapes.py

```diff
@@ -3203,7 +3208,9 @@ def issue_guard(guard: ShapeGuard) -> None:
 
         # First, issue all the non-trivial guards.
         for guard in self.guards:
-            if self._maybe_evaluate_static(guard.expr) is not None:
+            static_expr = self._maybe_evaluate_static(guard.expr)
+            if static_expr is not None:
+                self.log.debug("eval %s == %s [statically known]", guard.expr, static_expr)
```

Comment:
Are you sure about this log? We don't log other `issue_guard` here. This seems like it could be confuisng.